pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -550,28 +540,50 @@ func generateProwjob(ciopConfig *api.ReleaseBuildConfiguration,
 		return nil, fmt.Errorf("BUG: test '%s' not found in injected config", inject.Test)
 	}
 
-	var refs []prowv1.Refs
+	prsByRepo := make(map[string][]v1.PullRequestUnderTest)
 	for _, pr := range prs {
-		refs = append(refs, prowv1.Refs{
-			Org:  pr.Org,
-			Repo: pr.Repo,
+		orgRepo := fmt.Sprintf("%s/%s", pr.Org, pr.Repo)
+		prsByRepo[orgRepo] = append(prsByRepo[orgRepo], pr)
```

Comment:
Do we assume that the user has been careful enough not to specify the same PR more than once, right?

Reply:
`clonerefs` is actually okay with that. It does do a second merge of the changes, but it ends up resulting in the same source as if it were only merged once. In the end, it just wastes a couple of seconds.